views/js/test/runner/plugins/probes/latency/test.js

```diff
@@ -232,7 +233,9 @@ define([
             getProxy: function() {
                 return {
                     sendVariables: function() {
-                        return new Promise();
+                        return new Promise(function (resolve) {
+                            resolve();
+                        });
```

Comment:
Since the promise is immediately resolving, can you simplify it by `Promise.resolve()`?